src/endpoints/KeyWrapper.ts

```diff
@@ -212,7 +217,7 @@ export class KeyWrapper {
   private static getEncryptedKeyMaterial(
     wrappingKey: ArrayBuffer | undefined,
     payload: IKeyItem,
-  ): [string, string] {
+  ): [string, string | undefined] {
     Logger.debug(`getEncryptedKeyMaterial: `, payload);
     const receipt = payload.receipt;
     delete payload.receipt;
```

Comment:
What is the condition that `payload.receipt` is undefined? I'm wondering if we should allow undefined for the response?

Reply:
if receipt is undefined, the delete operation has no effect.

Reply:
Thanks for letting me know that, but my question is really:
Is it possible getEncryptedKeyMaterial is called with `payload.receipt === undefined`? If so when is it?

I'm asking this because return type `[string, string]` is better than `[string, string | undefined]` if possible. Caller wouldn't have to handle undefined.

Reply:
I found the answer:

In key(), you can find:

```
// Get receipt if available
  if (receipt !== undefined) {
    keyItem.receipt = receipt;
    Logger.debug(`Key->Receipt: ${receipt}`);
  } else {
    return ServiceResult.Accepted();
  }
```.

So both `payload.receipt === undefined` and `payload.receipt !== undefined`, is expected.